app/modules/audit_logs/parameters.py

```diff
@@ -8,7 +8,13 @@
 from app.extensions.api.parameters import PaginationParameters
 
 
-class GetCollaborationParameters(PaginationParameters):
+class GetAuditLogParameters(PaginationParameters):
```

Comment:
See, you guys failed the test I set to see if you spotted this :rofl:

Reply:
Well either I set a booby trap to see if you were paying attention or I missed it myself and spotted it second time around. 
Go on, have a guess which is most likely :rofl: